frame/vamm/src/lib.rs

```diff
@@ -270,25 +278,24 @@ pub mod pallet {
 		/// effect at the time `timestamp`.
 		pub closed: Option<Moment>,
 
-		/// The time weighted average price for
-		/// [`base_asset_reserves`](VammState::base_asset_reserves).
-		pub base_asset_twap: Balance,
-
-		/// The time weighted average price timestamp for
-		/// [`base_asset_reserves`](VammState::base_asset_reserves).
-		pub base_asset_twap_timestamp: Moment,
+		/// The time weighted average price of
```

Comment:
I would say "of  **the** [`base`](composable_traits::vamm::AssetType::Base) asset w.r.t. **the** [`quote`](composable_traits::vamm::AssetType::Quote) asset." here and below

Reply:
Thanks! I'll change it in the next PR